app/components/page.tsx

```diff
@@ -24,9 +73,35 @@ interface PageHeaderProps extends React.HTMLAttributes<HTMLDivElement> {
 
 export function PageHeader({ children, className, ...props }: PageHeaderProps) {
   return (
-    <div className={cn('flex flex-col gap-2 p-2', className)} {...props}>
-      {children}
-    </div>
+    <header
```

Comment:
still needs some work to support back buttons and more flexible placement within the header